compiler/src/iree/compiler/Codegen/LLVMCPU/ConvertToLLVM.cpp

```diff
@@ -433,6 +433,12 @@ class HALDispatchABI {
           loc, basePtrValue.getType(), basePtrValue, baseOffsetValue);
     }
 
+    // TODO: This ignores memrefType.getMemorySpace, which is a HAL descriptor
```

Comment:
We chatted the other day but calling it out for others so they can think about it too (nothing actionable, just FYI): even when targeting CPU we may need to handle different memory spaces in harvard architecture systems and we can use this to do that - the host portion would decide to place memory for example in ROM/SRAM/etc, set the descriptor type appropriately, and then we'll need to ensure LLVM emits the appropriate loads/stores. Mostly only impacts the embedded space (AVR and various tiny RISCV systems being the most popular public examples) but is relevant on other weirder architectures too (TPUs especially). Hopefully we can put that off for awhile :)